hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/MergeOnReadSnapshotRelation.scala

```diff
@@ -56,7 +58,7 @@ class MergeOnReadSnapshotRelation(val sqlContext: SQLContext,
                                   val userSchema: StructType,
                                   val globPaths: Seq[Path],
                                   val metaClient: HoodieTableMetaClient)
-  extends BaseRelation with PrunedFilteredScan with Logging {
+  extends BaseRelation with CatalystScan with Logging {
```

Comment:
@garyli1019 : what's the counter part of this file for COW? I understand this PR is one for MOR. but just for my understanding.

Reply:
we don't have an independent class counterpart for COW since we just need to pass a filter to a normal parquet reader. 
https://github.com/apache/hudi/blob/master/hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/DefaultSource.scala#L165